_modules/dotnet-clean.test.js

```diff
@@ -0,0 +1,127 @@
+/**************************************************************************************************
```

Comment:
👏 nice job testing this files. asserting shell exits are occurring is great and important